src/xrpld/app/main/GRPCServer.cpp

```diff
@@ -574,11 +576,21 @@ GRPCServerImpl::start()
     cq_ = builder.AddCompletionQueue();
     // Finally assemble the server.
     server_ = builder.BuildAndStart();
+    serverPort_ = static_cast<std::uint16_t>(port);
 
-    return true;
+    return static_cast<bool>(serverPort_);
 }
 
-void
+boost::asio::ip::tcp::endpoint
+GRPCServerImpl::getEndpoint() const
+{
+    std::string const addr =
+        serverAddress_.substr(0, serverAddress_.find_last_of(':'));
+    return boost::asio::ip::tcp::endpoint(
```

Comment:
Or you can put an explanation about handling port=0 here.

Reply:
comment adde to `evconfig.cpp`